t/rearrange_snp_matrix.t

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env perl
+use warnings;
+use strict;
+
```

Comment:
Allow for this test script to work with 'prove' when running from root directory.